### ISSUE TITLE
Make npc stop in place when mission is accepted 

### ISSUE THREAD

**Issue by baraba3** [CleverRaven/Cataclysm-DDA#16812]
It Is really annoying to chase them down  after you have finished mission they gave you..Especially when they run into woods as at least in fields you can catch with them in car but in woods its nearly impossible.

Several other notes/problems/exploits about npc i noticed
1.i dont know if its intended but if you ask them to join you and they decline you can never ask them again as they always say you asked me recently...
2.giving them high value items in trade (mutagens and guns and electronics) after depleting their cash will make them love you-100% chance of join-bribing them sure is good mechanic but i dont think its intended on this way
3.npc reset food and water needs when put on guard-i think its ok he dont have needs on guard but active service shoudl be taxed by food and water. and this is easily exploited
4.i need to test it more but some npc when put on guard tend to make items out of thin air and drop them beneath them.in my current game npc i put on guard have beneath him books,pistols,sledgahmmer,several tools....thre was not zombies in room and its lab so its impossible i already placed him on pile.although i  treat this as simulated independent scavenging lol

**Comment by mutt3x** [comment]
Hmm i remember the chasing from somewhere...
#13956

About the the whole chasing thing, yeah it is annoying. I dont think that just stoping and waiting on the current location is okey. Maybe go to a first house in sight or a bed maybe? Anything sheltered is more realistic i guess.

Also, perhaps implementing a shout that will stop the known npc for some xy time or rather that the npc starts running to you after the shout? Thats also okey but i cant say how effective it would be.

**Comment by baraba3** [comment]
i did not consider waiting on said location ideal but stopgap measure that seem not so hard to implement.
Getting frequency of radio ,them cowereing in nearby shelter or npc telling of location where to find him would off course be more preferable but idoubt thats easy to code.

**Comment by pisskop** [comment]
Id be happier if they agreed to meet you at a location, or told you where they were going.

Things like sewer plants are monster free, things like you sending them to the normally safe campsites could work.


One of my issues is why on Earth would a survivor just  . . . stand out in the rain?  Id be happier if they went somehwere and told you up front.

**Comment by baraba3** [comment]
currenyl they are standing in rain/snow when you are not round.when you are they are running away in one direction trying to kill anything in their way.i dont think they are going anywhere or have any plan.

if only overmap could function as diffrent reality buble in which npc /fungus/trffieds could do their moves...

**Comment by Vidiveni** [comment]
Would rather have the npcs move a bit more randomly, as now they seem to always be running straight away from you when trying to turn in the completed quest. Doesn't seem to matter from which direction I approach them with the reality bubble.

**Comment by Leland** [comment]
Stalled discussion – closing. If there's interest still, comment here and I can reopen it; or feel free to make a new issue.
